modules/drivers/lidar_surestar/BUILD

```diff
@@ -0,0 +1,53 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "libsurestar_driver_component.so",
+    linkopts = ["-shared"],
```

Comment:
Follow the convention in other BUILD files how the  `cc_binary` + `libxxx_component.so` rule should be written in the `master` branch.

Reply:
changed